sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample12_ExtractSummaryConvenience.cs

```diff
@@ -0,0 +1,113 @@
+                + " post-pandemic hybrid workforce of the future, she added. It enables employees accustomed to working from home to continue working from home; it enables"
+                + " companies to hire interns from halfway around the world; it allows startups to scale without requiring IT expertise. “I think this will be interesting for"
+                + " those organizations who, for whatever reason, have shied away from virtualization. This is giving them an opportunity to try it in a way that their regular,"
+                + " everyday endpoint admin could manage,” McKelvey said. The simplicity of Windows 365 won over Dean Wells, the corporate chief information officer for the"
+                + " Government of Nunavut. His team previously attempted to deploy a traditional virtual desktop infrastructure and found it inefficient and unsustainable given"
+                + " the limitations of low-bandwidth satellite internet and the constant need for IT staff to manage the network and infrastructure. We didn’t run it for very"
+                + " long,” he said. “It didn’t turn out the way we had hoped. So, we actually had terminated the project and rolled back out to just regular PCs.” He re-evaluated"
+                + " this decision after the Government of Nunavut was hit by a ransomware attack in November 2019 that took down everything from the phone system to the"
+                + " government’s servers. Microsoft helped rebuild the system, moving the government to Teams, SharePoint, OneDrive and Microsoft 365. Manchester’s team recruited"
+                + " the Government of Nunavut to pilot Windows 365. Wells was intrigued, especially by the ability to manage the elastic workforce securely and seamlessly. “The"
+                + " impact that I believe we are finding, and the impact that we’re going to find going forward, is being able to access specialists from outside the territory"
+                + " and organizations outside the territory to come in and help us with our projects, being able to get people on staff with us to help us deliver the day-to-day"
+                + " expertise that we need to run the government,” he said. “Being able to improve healthcare, being able to improve education, economic development is going to"
+                + " improve the quality of life in the communities.”";
+
+            // Prepare the input of the text analysis operation. You can add multiple documents to this list and
+            // perform the same operation on all of them simultaneously.
+            List<string> documentBatch = new()
+            {
+                document
+            };
+
+            // Start the text analysis operation.
+            ExtractSummaryOperation operation = client.StartExtractSummary(documentBatch);
+
+            // Wait for completion with manual polling.
+            TimeSpan pollingInterval = new(1000);
```

Comment:
just wondering why we show manual polling. does this still honor retry-after?

Reply:
This is just the pattern I saw in the other samples and I was simply following it. It does not honor retry-after, and I personally think it should be removed as it doesn't really add value. I'll start by removing it from the new samples I'm adding (i.e., extractive summarization, abstractive summarization, and dynamic classification).

I have created an issue to revisit the rest of the samples in the future: https://github.com/Azure/azure-sdk-for-net/issues/32735